components/sysutils/e2fsprogs/Makefile

```diff
@@ -0,0 +1,59 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+#
+# Copyright 2022 Stephan Althaus
+#
+
+BUILD_BITS=		64
+#BUILD_STYLE=	cmake
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		e2fsprogs
+COMPONENT_VERSION=	1.46.4
+#COMPONENT_REVISION=	1
+COMPONENT_SUMMARY=	Second extended file system (ext2fs) management programs
+COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.xz
+COMPONENT_ARCHIVE_HASH= \
+	sha256:b11042533c1b1dcf17512f0da48e05b0c573dada1dd8b762864d10f4dc399713
+# 1.46.2: 23aa093295c94e71ef1be490c4004871c5b01d216a8cb4d111fa6c0aac354168	
+# 1.46.4: b11042533c1b1dcf17512f0da48e05b0c573dada1dd8b762864d10f4dc399713
+COMPONENT_ARCHIVE_URL= \
```

Comment:
Why are you using a manual "version control"? Please don't leave commented out lines without good reasons and some additional comments.